Library/Homebrew/shims/scm/git

```diff
@@ -92,7 +91,7 @@ case "$(lowercase "$SCM_FILE")" in
     ;;
 esac
 
-brew_version="$(quiet_safe_cd "$SCM_DIR/../../../../bin" && pwd -P)/$SCM_FILE"
+brew_version="$(quiet_safe_cd "$HOMEBREW_PREFIX/bin" && pwd -P)/$SCM_FILE"
```

Comment:
Subprocesses may filter the environment and, as-is, this will break silently. We need to detect `HOMEBREW_PREFIX` even if this means duplicating the existing logic in `bin/brew`. Additionally, `shims/scm/svn` probably wants the same logic, here.